CONTRIBUTING.md

```diff
@@ -115,11 +115,11 @@ As soon as you have completed implementing your contributions, follow the steps
 
 First, make sure all test cases are passing.
 ```sh
-npm test
+npm run test
 ```
 Secondly, verify that the linter returns no errors.
 ```sh
-npm lint
+npm run check-lint
```

Comment:
Can you change this to `npm run check` to check for typescript errors as well? You could add that to the text above as well.